code/datums/mutantraces.dm

```diff
@@ -1102,6 +1134,14 @@
 	decomposes = FALSE
 	race_mutation = /datum/bioEffect/mutantrace/skeleton
 
+	emote(var/act, var/voluntary)
+		switch(act)
+			if ("rattle")
+				if (mob.emote_check(voluntary, 50))
+					. = list("<B>[mob]</B> rattles", "<i>rattles</i>")
```

Comment:
Might also be worth having the skeleton's speech sound play here as well. Really sells the rattle!